frontend/src/components/CategorySelect/CategorySelect.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+
+import { typedMemo } from 'src/core/utils/typed-memo';
+import { Category } from 'src/core/models/category';
+import { Typography } from 'antd';
+
+import './CategorySelect.scss';
+
+const { Text } = Typography;
+
+type Props = Readonly<{
+
+  /** Массив категорий навыков. */
+  categories: Category[];
```

Comment:
массив тоже можно сделать `readonly`, чтоб избежать мутаций
```suggestion
  categories: readonly Category[];
```